pyproject.toml

```diff
@@ -35,7 +35,10 @@ tracker = "https://github.com/daggaz/json-stream/issues"
 funding = "https://www.buymeacoffee.com/daggaz"
 
 [project.optional-dependencies]
-requests = ["requests"]
+requests = [
```

Comment:
shouldn't this be a second optional called `httpx`?

Reply:
Good catch. I don't know much about pyproject.toml (except what `poetry` automatically writes) but this is clearly boneheaded.